tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/TestHelper.cs

```diff
@@ -644,6 +646,62 @@ protected void EnableDirectLogSubmission(int intakePort, string integrationName,
             Assert.Equal(expectedServiceVersion, span.Tags.GetValueOrDefault(Tags.Version));
         }
 
+        protected async Task ReportRetry(ITestOutputHelper outputHelper, int attemptsRemaining, Type testType, Exception ex = null)
```

Comment:
LGTM, but you might want to consider moving this to the `CustomTestFramework`, and looking for a custom `[AutoRetry]` flag?

Reply:
It's not a lot of extra work I admit, but I'll keep it simple for now (simple for me, not for users :p)